README.md

```diff
@@ -1,63 +1,318 @@
+  <summary>Environment Configuration</summary>
+  <ul>
+    <li><a href="https://github.com/motdotla/dotenv">dotenv</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Monorepo Tools</summary>
+  <ul>
+    <li><a href="https://nx.dev/">Nx</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Code Quality</summary>
+  <ul>
+    <li><a href="https://github.com/pahen/madge">Madge</a></li>
+    <li><a href="https://github.com/davidmarkclements/trace-unhandled">trace-unhandled</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Containerization</summary>
+  <ul>
+    <li><a href="https://www.docker.com/">Docker</a></li>
+    <li><a href="https://docs.docker.com/compose/">Docker Compose</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>API Documentation</summary>
+  <ul>
+    <li><a href="https://swagger.io/">Swagger</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features
+- **Account creation using [SIWF](https://github.com/AmplicaLabs/siwf)**
+  - Includes behind the scenes delegation to the provider
+- **Get User and Provider Account(s)**
+- **Handle (aka username) claiming**
+  - Create
+  - Change
+- **Keys Handling**
+  - Add keys to an account
+  - Get keys
+- **Get delegation info for an msaId**
+- **Easy integration with Web3 wallets**
+
+<p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
+
+<!-- LIVE Docs -->
+
+## 🚀 Live Docs
+
+- [Live Docs](https://amplicalabs.github.io/{gateway-service-repo})
+
+<p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [Nodejs](https://nodejs.org)
+- [Docker](https://docs.docker.com/get-docker/)
+
+### Environment Variables
 Modify any environment variables in the `.env` file as needed. For docker compose env `.env.docker.dev` file is used. The complete set of environment variables is documented [here](./ENVIRONMENT.md), and a sample environment file is provided [here](./env.template)
 
-### 3. Start the service:
-Run the following command to start the service:
-```bash
-docker-compose up
+### ⚡ Quick Start
+
+In order to run the `account-service` in development mode without containers, you can use the following commands:
+
+1. Start the Redis server container and the Frequency container. You can view the logs with your Docker setup.
+
+   ```bash
+   docker-compose up -d redis frequency
+   ```
+
+2. In a new terminal window, start the `account-service` api app. Logs will be displayed in the terminal for easy reference.
+
+    ```bash
+    npm run start:api:debug
+    ```
+
+3. In another terminal window, start the `account-service` worker app.
+
+    ```bash
+    npm run start:worker:debug
+    ```
+   
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  git clone git@github.com:AmplicaLabs/account-service.git
+  cd account-service
```

Comment:
I would nest this under "Quick Start", as the first step

Reply:
I just moved this and npm install above quickstart.